src/app/core/services/rundown-event-observer.service.ts

```diff
@@ -61,9 +61,9 @@ export class RundownEventObserver {
     return this.eventObserver.subscribe(RundownEventType.SET_NEXT, this.createEventValidatingConsumer(onSetNext, this.rundownEventParser.parseSetNextEvent.bind(this.rundownEventParser)))
   }
 
-  public subscribeToRundownInfinitePieceAdded(onInfinitePieceAdded: (event: RundownInfinitePieceAddedEvent) => void): EventSubscription {
+  public subscribeToRundownInfinitePieceAdded(onInfinitePieceAdded: (event: RundownInfinitePiecesUpdatedEvent) => void): EventSubscription {
```

Comment:
Parameter name should be updated as well.

Reply:
I can't believe i missed all these, thanks!